product_docs/docs/biganimal/release/reference/cli/using_features.mdx

```diff
@@ -102,12 +102,12 @@ See [Other monitoring and logging solutions](/biganimal/latest/using_cluster/05_
 
 ## Maintenance windows CLI command
 
-You can set and view maintenance windows using the `cluster-maintenance-window` CLI command.
+You can set and view maintenance windows using the `cluster set-maintenance-window` CLI command.
```

Comment:
You can set using: `cluster set-maintenance-window`
view using: `cluster get-maintenance-window`